google/cloud/forseti/services/explain/explainer.py

```diff
@@ -197,7 +295,14 @@ def get_permissions_by_roles(self, model_name, role_names, role_prefixes):
                 yield result
 
     def denormalize(self, model_name):
-        """Denormalizes a model."""
+        """Denormalizes a model.
```

Comment:
It is not clear what `denormalize` means.  So, it might deserves more explanation / documentation what it is suppose to do?  Either here or closer to the core of the denormalize()

Reply:
done